crates/bevy_pbr/src/render_graph/lights_node.rs

```diff
@@ -75,19 +76,34 @@ pub struct LightsNodeSystemState {
     max_lights: usize,
 }
 
+// Ambient color.
+#[derive(Debug)]
+pub struct AmbientLight {
```

Comment:
Can we move this to `light.rs` and make sure its exported at `bevy_pbr::AmbientLight`?

Reply:
Done

Reply:
Wait, I missed the pub(crate).

Reply:
Something weird is happening where my latest commit isn't being added here. I think I may create a new PR.

Reply:
It all gets squashed down in the end, so thats not a problem!